main.tf

```diff
@@ -160,4 +160,19 @@ resource "aws_cloudwatch_log_group" "log_group" {
   name              = "/aws/lambda/${aws_lambda_function.logs_to_datadog.function_name}"
   retention_in_days = var.retention
   tags              = local.tags
-}
\ No newline at end of file
+}
+
+resource "aws_sns_topic_subscription" "sns_topic_arn" {
+  topic_arn = var.environment_alerts
```

Comment:
Did we want this to be `var.sns_topic_arn`?

Reply:
Yes, that was a temporary name used while I was working, and I forgot to change it. It has been changed now.